ui/app/components/input-number.js

```diff
@@ -1,56 +1,52 @@
 const Component = require('react').Component
 const h = require('react-hyperscript')
 const inherits = require('util').inherits
+const { addCurrencies } = require('../conversion-util')
 
 module.exports = InputNumber
 
 inherits(InputNumber, Component)
 function InputNumber () {
   Component.call(this)
 
-  this.state = {
-    value: 0,
-  }
-
   this.setValue = this.setValue.bind(this)
 }
 
 InputNumber.prototype.componentWillMount = function () {
-  const { initValue = 0 } = this.props
+  const { value = 0 } = this.props
 
-  this.setState({ value: initValue })
+  this.setState({ value: Number(value) })
```

Comment:
If we're leaving state in this component (which I think is okay), why do we need L33?

Reply:
I meant to take state out of this component. Fixed in 32098a64